includes/Modules/Analytics.php

```diff
@@ -1379,14 +1418,19 @@ protected function create_analytics_site_data_request( array $args = array() ) {
 			$request->setDateRanges( array( $date_range ) );
 		}
 
+		if ( ! empty( $args['dimension_filters'] ) ) {
+			$request->setDimensions( (array) $args['dimension_filters'] );
+		}
+
```

Comment:
@ivankruchkoff do we really need to set dimensions here using `$args['dimension_filters']`? We already set dimensions using `$args['dimensions']` a few steps above.

Reply:
Removed.